resources/synapse_admin.rb

```diff
@@ -0,0 +1,36 @@
+# Synapse Admin Web Application
+# This resource allows for the deployment of an Admin web-interface for Matrix Synapse servers.
+# NOTE: Requires /_synapse/admin to be reachable from the browser
+
+resource_name :osl_synapse_admin
+provides :osl_synapse_admin
+unified_mode true
+
+default_action :create
+
+property :matrix_domain, String, name_property: true
+property :port, Integer, default: 8080
+property :force_domain, [true, false], default: false
```

Comment:
Let's match the name that upstream uses. I'd recommend we force this to true by default.
```suggestion
property :restrict_base_url, [true, false], default: true
```